apps/api/internal/apperrors/error_test.go

```
@@ -0,0 +1,136 @@
+	var response Error
+	if err := json.Unmarshal(recorder.Body.Bytes(), &response); err != nil {
+		t.Fatalf("response is not valid JSON: %v", err)
+	}
+	if response.ErrCode != string(Unknown) || response.Message != "internal process failed" {
+		t.Errorf("response = %+v", response)
+	}
+}
+
+func TestErrorHandlerWithoutCause(t *testing.T) {
+	recorder := httptest.NewRecorder()
+	req := httptest.NewRequest(http.MethodGet, "/profile", nil)
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)